.clusterfuzzlite/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# c-toxcore Clusterfuzzlite build environment
+
+# We want to use the latest tools always
+# hadolint ignore=DL3006
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update && \
+    apt-get -y install --no-install-suggests --no-install-recommends \
+    cmake libtool autoconf automake pkg-config \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+
+RUN git clone --depth 1 --branch 1.0.18 https://github.com/jedisct1/libsodium libsodium
```

Comment:
Add a `.hadolint.yml` in the repo root dir, containing a suppression for DL3059 (see https://github.com/TokTok/dockerfiles/blob/master/.hadolint.yaml#L6).

Reply:
added to existing file